src/math/Ray.js

```diff
@@ -220,12 +220,13 @@ class Ray {
 	intersectSphere( sphere, target ) {
 
 		_vector.subVectors( sphere.center, this.origin );
-		const tca = _vector.dot( this.direction );
+
 		const d2 = _vector.dot( _vector ) - tca * tca;
```

Comment:
`tca` is used on this line.

Reply:
Indeed. I can't say I understand why, though. But judging from my experience today, it seems like a good idea to not challenge it right away... 😳🤔